src/main/java/org/desarrolladorslp/technovation/services/EventService.java

```diff
@@ -0,0 +1,11 @@
+package org.desarrolladorslp.technovation.services;
+
+import org.desarrolladorslp.technovation.controller.dto.EventDTO;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface EventService {
+
+    List<EventDTO> listEvents(int year, int month, UUID userId);
```

Comment:
Notice that you use `UUID userId, int year, int month` in BatchService... Not requieres, but try to use the same order un parameters if they are the same thing